buildspecs/linux_x86-64.spec

```diff
@@ -101,7 +101,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<property name="uma_make_cmd_cxx_exe_ld" value="$(CC)"/>
 		<property name="uma_make_cmd_dll_ld" value="$(CC)"/>
 		<property name="uma_make_cmd_exe_ld" value="$(CC)"/>
-		<property name="uma_make_cmd_interp_gcc" value="gcc-4.6"/>
```

Comment:
We can just remove this variable, and the places that use it, in mkconstants.mk.ftl line 44 and targets.mk.inux.inc.ftl line 511

Reply:
done